docs/guides/actions.md

```diff
@@ -631,7 +631,7 @@ This is analogous to the SCXML `<if>`, `<elseif>`, and `<else>` elements: [www.w
 
 ## Pure Action
 
-The `pure()` action creator is a pure function (hence the name) that returns the action object(s) to be executed based on the current state `context` and `event` that triggered the action. This allows you to dynamically define which actions should be executed.
+The `pure()` action creator is a pure function (hence the name) that returns the action object(s) to be executed based on the current state `context` and `event` that triggered the action. It allows you to define which actions to perform dynamically.
```

Comment:
This is a bit more confusing than the original. The actions aren't "performed dynamically", they're dynamically defined.